doc/whats_new/v1.4.rst

```diff
@@ -40,3 +40,11 @@ Thanks to everyone who has contributed to the maintenance and improvement of
 the project since version 1.3, including:
 
 TODO: update at the time of the release.
+
+:mod:`sklearn.base`
+...................
+
+- |Enhancement| :func:`base.ClusterMixin.fit_predict` and
+  :func:`base.OutlierMixin.fit_predict` now accept ``**kwargs`` which are
```

Comment:
`OutlierMixin.fit_predict` linking is broken for some reason. I think that we should also use `:meth:` instead of `:func:` (but `:func:` should still work).